portfolio/src/main/webapp/experience.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <style>
+ul {
+  background-color: #333;
+  list-style-type: none;
+  margin: 0;
+  overflow: hidden;
+  padding: 0;
+}
+
+li {
+  float: left;
+}
+
+li a {
+  color: white;
+  display: block;
+  padding: 14px 16px;
+  text-align: center;
+  text-decoration: none;
+}
+
+li a:hover:not(.active) {
+  background-color: #111;
+}
+
+.active {
+  background-color: #4CAF50;
+}
+
+</style>
+</head>
+<body>
+
+<ul>
+  <div id="homepage">
+    <li><a href="./index.html">Home</a></li>
+  </div>
+
+   <div id="about">
+    <li><a href="./about.html">About Me</a></li>
+   </div>
+
+   <div id="experience">
+    <li><a class="active" href="./experience.html">Experience</a></li>   
+   </div>
+</ul>
```

Comment:
I guess you are repeating this so that the navbar appears in each page. Ok for now. But look for a so you don't repeat the navbar code in each of the html file.

html5 has a nav tag - https://www.w3.org/community/webed/wiki/Creating_multiple_pages_with_navigation_menus

Reply:
Will implement in a later commit (done for now!)